### PR TITLE
Get Samples: first bulk function

### PR DESCRIPTION
**Pull request by slebras**

### CODE REVIEW

File: lib/SampleService/core/api_translation.py [hunk 1]
```diff
@@ -325,6 +325,25 @@ def get_sample_address_from_object(
...
+    :returns: a tuple containing the ID and the version or None if no version was provided.
+    :raises MissingParameterError: if the ID is missing or the version is required and not present.
+    :raises IllegalParameterError: if the ID is malformed or if the version is not an
+        integer or < 1.
+    '''
+    return [
+        {"id": _cast(UUID, get_id_from_object(samp_obj, ID, required=True)),
+        "version":get_version_from_object(samp_obj, version_required)
+        }
+        for samp_obj in params['samples']
```
Comment (Tianhao-Gu) [inline lib/SampleService/core/api_translation.py:345]: why not reuse `get_sample_address_from_object`?

File: lib/SampleService/core/samples.py [hunk 2]
```diff
@@ -186,6 +186,17 @@ def get_sample(
         self._check_perms(_not_falsy(id_, 'id_'), user, _SampleAccessType.READ, as_admin=as_admin)
         return self._storage.get_sample(id_, version)
 
+    def get_samples(
```
Comment (Tianhao-Gu) [inline lib/SampleService/core/samples.py:189]: same question here. I think by reusing `get_sample` here, we can save all updates in `arango_sample_storage.py`.

**Review by Tianhao-Gu — APPROVED** [review]
overall looks good. But I think there are some functions that can be reused. I think if you just reuse `get_sample` in the main `SampleServiceImpl`, it will waste `_check_admin` multiple times. But makes future maintenance a lot easier.  But I think we could certainly reuse `get_sample_address_from_object` and `get_sample` in `samples.py`.

**Comment by zhlu9890** [comment]
It looks good to me.

**Review by Tianhao-Gu — APPROVED** [review]
👍 LGTM

**Pull request merged**